back/src/adapters/primary/routers/searchImmersion/createSearchImmersionRouter.ts

```diff
@@ -30,5 +30,13 @@ export const createSearchImmersionRouter = (deps: AppDependencies) => {
     ),
   );
 
+  expressSharedRouter.getImmersionOffer(async (req, res) =>
+    sendHttpResponse(req, res, () =>
+      deps.useCases.getSearchImmersionResultBySiretAndAppellationCode.execute(
+        req.query,
+      ),
+    ),
+  );
```

Comment:
le nom de la route ne m'a pas l'air correcte, et celui du usecase peut-être pas nom plus :

```suggestion
  expressSharedRouter.getSearchResult(async (req, res) =>
    sendHttpResponse(req, res, () =>
      deps.useCases.getSearchResultBySiretAndAppellationCode.execute(
        req.query,
      ),
    ),
  );
```